addons/l10n_eg_edi_eta/models/account_edi_format.py

```diff
@@ -253,7 +253,7 @@ def _l10n_eg_eta_prepare_invoice_lines_data(self, invoice, tax_data):
             discount_amount = self._l10n_eg_edi_round(price_subtotal_before_discount - abs(line.balance))
             item_code = line.product_id.l10n_eg_eta_code or line.product_id.barcode
             lines.append({
-                'description': line.name,
+                'description': line.product_id.display_name,
```

Comment:
Shouldn't be something like this?
```suggestion
                'description': line.product_id.display_name or line.name,
```
What's happened if we don't have product?